spec/rubocop/cop/rspec/indexed_let_spec.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::IndexedLet do
+  let(:cop_config) do
+    {
+      'MaxRepeats' => 2,
+      'LetTypes' => ['let']
+    }
+  end
+
+  context 'when inside describe' do
+    context 'when file contains one indexed let' do
+      specify do
+        expect_no_offenses(<<~RUBY)
+          describe SomeService do
+            let(:item_1) { create(:item) }
+          end
+        RUBY
+      end
+    end
+
+    context 'when file contains maximum indexed repeats' do
+      specify do
+        expect_offense(<<~RUBY)
+          describe SomeService do
+          ^^^^^^^^^^^^^^^^^^^^^^^ This block declares indexed `let` statements: `item_x`. Please give them meaningful names, use create_list or move creation to the `before` block.
+            let(:item_1) { create(:item) }
+            let(:item_2) { create(:item) }
```

Comment:
Would it catch indexes not prefixed with an underscore, e.g. `item1`/`item2`? Why not?

Reply:
Because I forgot it 😅now this check is in place too